### ISSUE TITLE
Please add .command() function a path config

### ISSUE THREAD

**Issue by winys** [tj/commander.js#481]
It would be better if we can customize the path of sub-command file.

**Comment by szdavid92** [comment]
Our naming policy enforces camel case for filenames, but subcommands are more naturally written in kebab case. Currently commander doesn't do this mapping neither gives an option to for me to do it manually. It would be great if there would be either

- camelize option for subcommands or
- a way to configure the executables path.

**Comment by shadowspawn** [comment]
Pull Request #532 adds an option for this.

I am interested in feedback on `executableFile` as the option name?

e.g.
```
program
.command("example", "example description", { executableFile: "myExampleCommand" });
```

**Comment by shadowspawn** [comment]
Pull Request being considered for v3 is #999

**Comment by shadowspawn** [comment]
This issue will be resolved when v3.0.0 is released. Available now as a prerelease. See #1001

**Comment by shadowspawn** [comment]
`.command` now supports the `executableFile` option (thanks to @abetomo). 
Shipped in v3: https://github.com/tj/commander.js/releases/tag/v3.0.0
